### PR TITLE
tvseries plugin: tell when series are cancelled

### PR DESCRIPTION
**Pull request by moopie**
hi :)

### CODE REVIEW

**Comment by moopie** [comment]
Though i guess 'cancelled' is not correct in all cases

**Comment by EricBisceglia** [comment]
> hi :)

Hi.

File: plugins/tvseries.py [hunk 1]
```diff
@@ -53,7 +53,11 @@ def get_next_episode_info(show, output_timezone=pytz.timezone('UTC')):
 			log.debug('time left: %r (%s)', time_left, time_left_str)
 			info += ' (in %s)' % time_left_str
 	else:
-		info += ' - no next episode :('
+		status = data['status']
+		if status == 'Ended':
+			info += ' - cancelled :('
+		else:
```
Comment (EricBisceglia) [inline plugins/tvseries.py]: Nice usage of python ternary but split into vertical lines

Comment (moopie) [reply plugins/tvseries.py]: pyton ternary is pretty shit

**Comment by anlutro** [comment]
We'll try it out, if it's wrong too many times we can change it later on

File: plugins/tvseries.py (hunk 1 above)
Comment (phixion) [reply plugins/tvseries.py]: 乇乂ㄒ尺卂 ㄒ卄丨匚匚

**Pull request merged**